### PR TITLE
Some small updates to the landing page

### PR DESCRIPTION
**Pull request by fangerer**
Addresses https://github.com/hpyproject/hpy/issues/342 .

I tried to remove the *alpha* impression from the website. My manager was looking at it and immediately had a negative impression.

Also, I've added a link to the Discord server.

### CODE REVIEW

File: pages/index.rst [hunk 1]
```diff
@@ -45,44 +45,33 @@ What are the advantages of HPy?
...
 .. _`PEP 620`: https://peps.python.org/pep-0620/
 
-Current status and where we need help
-=====================================
+Current status
+==============
 
-HPy is in development. `0.0.4`__ is the latest alpha release.
+HPy is under active development. `0.0.4`__ is the latest alpha release but we
+will soon leave the _alpha_ state and working hard towards a stable release.
```
Comment (timfel) [inline pages/index.rst:57]:
```suggestion
will soon leave the _alpha_ state and are working hard towards a stable release.
```

File: pages/index.rst [hunk 2]
```diff
@@ -45,44 +45,33 @@ What are the advantages of HPy?
...
 
 .. __: https://hpyproject.org/blog/posts/2022/06/hpy-0.0.4-third-public-release/
 
-We welcome your design input or adventurous alpha testing.
+In the last year, we invested a lot of effort in adding more APIs by migration
```
Comment (timfel) [inline pages/index.rst:61]:
```suggestion
We feel that the HPy ABI is stable enough now that we can fulfill our backwards and forwards binary compatibility promises with the upcoming release, and that the API now covers enough use cases to migrate
```

File: pages/index.rst [hunk 3]
```diff
@@ -45,44 +45,33 @@ What are the advantages of HPy?
...
+important packages (in particular, checkout our numpy_ port). We also improved
```
Comment (timfel) [inline pages/index.rst:62]:
```suggestion
important packages (in particular, checkout our numpy_ port).
```

File: pages/index.rst [hunk 4]
```diff
@@ -45,44 +45,33 @@ What are the advantages of HPy?
...
+our documentation a lot (in particular the `API reference`_).
```
Comment (timfel) [inline pages/index.rst:63]:
```suggestion
We also provide porting guides and extensive documentation (in particular the `API reference`_).
```

File: pages/index.rst [hunk 5]
```diff
@@ -95,8 +84,8 @@ The extensions that we are experimenting with include:
   course).
 
 - numpy_: one of our ambitious goals is to port numpy to HPy, and to use this
-  experience to better understand how to design the API. This builds but does
-  not pass the test suite.
+  experience to better understand how to design the API. This port are close to
```
Comment (timfel) [inline pages/index.rst:87]:
```suggestion
  experience to better understand how to design the API. This port is close to
```

File: pages/index.rst [hunk 6]
```diff
@@ -118,12 +109,39 @@ performance with GraalVM Python for the kiwi-solver complete HPy port.
 .. _`kiwi-solver`: https://github.com/hpyproject/kiwi-hpy/
 .. _`GraalVM blog post`: https://medium.com/graalvm/porting-matplotlib-from-c-api-to-hpy-aa32faa1f0b5
 
-This website and the logo are ugly
-===================================
+Where we need help
+==================
+
+* **Documentation:** Our resources are very limied and we therefore concentrate
```
Comment (paugier) [inline pages/index.rst:115]:
```suggestion
* **Documentation:** Our resources are very limited and we therefore concentrate
```

File: pages/index.rst [hunk 7]
```diff
@@ -118,12 +109,39 @@ performance with GraalVM Python for the kiwi-solver complete HPy port.
...
-We know: we have expertise in Virtual Machines, Compilers, low-level
-programming, etc, but zero expertise in web development or graphic
-design. `PRs are welcome ;) <https://github.com/hpyproject/hpyproject.org/>`_
+* **Publicity:** HPy is already well received among core developers of some
+  important projects. For instance, we are actively talking to Python, NumPy,
+  and Cython core developers. However, we need to get more package developers to
+  be interested in and even use HPy. We appreciate any help for advertising HPy
+  and we are also happy to help preparing talks, demos, etc.
+
+* **Tooling:** This is  also a topic we would need help. We are, for example,
```
Comment (paugier) [inline pages/index.rst:126]:
```suggestion
* **Tooling:** This is also a topic we would need help. We are, for example,
```

File: pages/index.rst (hunk 6 above)
Comment (fangerer) [reply pages/index.rst]: thanks for the catch

File: pages/index.rst [hunk 8]
```diff
@@ -95,8 +86,8 @@ The extensions that we are experimenting with include:
   course).
 
 - numpy_: one of our ambitious goals is to port numpy to HPy, and to use this
-  experience to better understand how to design the API. This builds but does
-  not pass the test suite.
+  experience to better understand how to design the API. This port is close to
+  pass the test suite.
```
Comment (hodgestar) [inline pages/index.rst:90]:
```suggestion
  passing the test suite.
```

**Review by hodgestar — APPROVED** [review]
I think this is a big improvement.

We should do a more thorough rewrite once we're a bit closer to 1.0.

**Pull request merged**